pages/map.tsx

```diff
@@ -41,7 +48,7 @@ export default function Home() {
   //This is a useEffect used to make sure that the users location is grabbed only once when the page is rendered
   useEffect(() => {
     if (!hasLoaded) {
-      setUserLocation()
+      setUserLocation(null)
```

Comment:
Why set to null?

Reply:
the setUserLocation has a bad name tbh. It is not a state it is a function that sets the map location, it has to have an arg or it will scream at me so that's why it is null so that it goes to either their gelocation or the default location(which I believe is london).